ang/civicase/case/search/directives/search.directive.js

```diff
@@ -43,6 +43,7 @@
 
     $scope.caseRelationshipOptions = caseRelationshipConfig;
     $scope.caseStatusOptions = _.map(caseStatuses, mapSelectOptions);
+    $scope.caseTypeOptions = [];
```

Comment:
Where is this used?

Reply:
Later on the file this is transformed by the `setCaseTypesBasedOnCategory` when the component is initialised. I just want to define all exposed variables on the top of the file so we are aware of them.